nginx-proxy/nginx.conf

```diff
@@ -14,7 +14,7 @@ events {
 
 http {
     include       /etc/nginx/mime.types;
-    default_type  application/octet-stream;
+    default_type  "";
```

Comment:
is this necessary?

Reply:
Without this nginx returns Content-type header twice.

Reply:
But it worked before, without returning two headers... right?

Reply:
Yes, before it returned only one Content-type header because $format variable was empty I think.  

https://github.com/MinimalCompact/thumbor/blob/b2246ce3226a1e8dddba21615ebe13e3d896dfbb/nginx-proxy/nginx.tmpl#L54